### PR TITLE
Add CRC-16 generator

### PR DESCRIPTION
**Pull request by CodeSmell**
This addresses issue #18

### CODE REVIEW

File: src/test/java/com/walmartlabs/x12/checksum/Crc16Test.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    public void test_generateCyclicRedundancyCheck_123456789() {
+        String blockText = "123456789";
+        String crcValue = crcUtil.generateCyclicRedundancyCheck(blockText);
+        assertNotNull(crcValue);
+        assertEquals("BB3D", crcValue);
+    }
+
+    @Test
+    public void test_verifyBlockOfText_null() {
+        String crcValue = null;
```
Comment (patelch) [inline src/test/java/com/walmartlabs/x12/checksum/Crc16Test.java]: Should we add a test case for the `verifyBlockOfText()` method where only the crcValue or blockText is null?
